### ISSUE TITLE
Error when synchronizing post on Firefox Developer Edition

### ISSUE THREAD

**Issue by jabby** [HubPress/hubpress.io#538]
Hi @anthonny 

I have a little problem with the last update of hubpress.io (0.9.1). After loggin I have no post. So I use the synchronize button. But when I click the synchronize button, there is an error.  I got this in the console : 

````
e.attributes.$$smap is undefined  app.3e89e3d9c935e845e2fd.js:1:89043
too much recursion[En savoir plus]  vendor.099e66f2e693b444d03c.js:2:1665655
La mémoire consommée par will-change est trop importante. La limite du budget correspond à la surface multipliée par 3 (1914240 px). Les occurrences de will-change dépassant le budget seront ignorées.
````

Sorry the message is in french 🤣 

Version of the browser: 
* Firefox Developer Edition 61.0b3

**Comment by jabby** [comment]
Maybe I habe a biggest problem than I thought. 
The synchronization is not available on chrome / ms edge. I get the following error

````
Unable to get property 'imagesdir' of undefined or null reference
````

Maybe I need to update my posts manually then synchronize again.

**Comment by anthonny** [comment]
It's a bug with the `:hp-image:` attribute when it refers a image stored in the `images` directory

IE https://github.com/jabbytechnologies/blog/edit/gh-pages/_posts/2015-04-13-Pourquoi-je-suis-passe-de-Wordpress-a-HubPress.adoc

I will fix that now.

FYI you do not need to specify the `images` folder, it is the `imagedir` folder in our asciidoctor configuration:
```
:hp-image: 2015/04/hubpress-login.png
```

**Comment by anthonny** [comment]
You can use this workaround:
```
:hp-image: https://blog.jabby-techs.fr/images/2015/04/hubpress-login.png
```

**Comment by anthonny** [comment]
Just released the 0.9.2 it should be good now
